encord/orm/label_row.py

```diff
@@ -180,11 +188,17 @@ class LabelRow(base_orm.BaseORM):
             ("classification_answers", dict),
             ("object_actions", dict),
             ("label_status", str),
+            ("export_history", list),
+            ("export_hash", str),
+            ("exported_at", str),
         ]
     )
 
     NON_UPDATABLE_FIELDS = {
         "label_hash",
+        "export_history",
```

Comment:
key_hash?

Reply:
Not sure what do you mean here. `key_hash` is a field of individual "history items", and we're only dealing with top-level fields of a label_row here.